Dockerfile

```diff
@@ -3,6 +3,12 @@ FROM node:10-alpine
 WORKDIR /build
 COPY . ./
 
+ARG BACKEND_API_URL
+ENV API_URL=${BACKEND_API_URL}
```

Comment:
How about: BACKEND_API_URL=${BACKEND_API_URL} 

same with GEOSERVER_API_URL

Reply:
I tried this, it doesn't work. I wanted it to work, It doesn't. I don't know why.

Reply:
How about:
ENV BACKEND_API_URL=${_BACKEND_API_URL}?

Reply:
These are the same variables that are used when building the docker image.
I don't think that the command: docker build --build-arg _BACKEND_API_URL="asd" .
complies to normal env variable standards

Reply:
Didn't quite understand why API_URL=${BACKEND_API_URL} works but ENV BACKEND_API_URL=${_BACKEND_API_URL} doesn't (added an extra _ to make those variables different from each other).